src/common/constants.ts

```diff
@@ -1,4 +1,5 @@
 const Constants = {
+  MaxCharCount: 2000,
```

Comment:
nit: i feel `maxMessageLength` is better

Reply:
Done